atlasdb-feign/src/main/java/com/palantir/atlasdb/http/FailoverFeignTarget.java

```diff
@@ -113,8 +114,7 @@ public void continueOrPropagate(RetryableException ex) {
                 // We implement some randomness around the expected value of BACKOFF_BEFORE_ROUND_ROBIN_RETRY_MILLIS.
                 // Even though this is not exponential backoff, should be enough to avoid a thundering herd problem.
                 long pauseTimeWithJitter = ThreadLocalRandom.current()
-                        .nextLong(BACKOFF_BEFORE_ROUND_ROBIN_RETRY_MILLIS / 2,
-                                (BACKOFF_BEFORE_ROUND_ROBIN_RETRY_MILLIS * 3) / 2);
+                        .nextLong(MAX_BACKOFF_BEFORE_ROUND_ROBIN_RETRY.toMillis());
```

Comment:
OK. I think this is fine as the point is only to avoid a thundering herd problem as mentioned.

nit: The comment should be updated